index.html

```diff
@@ -13,20 +13,53 @@
     .attribution { font-size: 11px; text-align: center; }
     .attribution a { color: hsl(228, 45%, 44%); }
   </style>
+  <link rel="stylesheet" href="./styles.css">
+  <!-- font 1 -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap" rel="stylesheet">
+  <!-- font 2 -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Display:wght@700&display=swap" rel="stylesheet">
 </head>
 <body>
-  
-  Sedans
-  Choose a sedan for its affordability and excellent fuel economy. Ideal for cruising in the city 
-  or on your next road trip.
+  <main>
+    <div class="columns">
+  <div class="column-container column-sedans">
+    <picture class="icon">
+      <img src="./icon-sedans.svg" alt="sedan icon">
```

Comment:
Very good for web accessibility that you included an alt attribute on the img element. Note that it is not necessary to add "icon" as part of the description since screen readers or similar assistive tech will read out the element type. For example, with this alt text it will read out "Sedan icon image". 

```suggestion
      <img src="./icon-sedans.svg" alt="sedan">
```

If you are curious about assistive tech and want to test your website with it, I recommend trying out enabling VoiceOver on Mac or installing JAWS on Windows. Alternatively, there are extensions for Chrome that you can enable, but they are not as widely used.

Reply:
What is assistive tech and how is it related to the "icon" tag? And what would VoiceOver/JAWS interact?

Reply:
**What is assistive tech?**
Assistive technology is a program, device or system that helps people with disabilities. For example, users with visual impairment or low vision can use a screen reader such as VoiceOver/JAWS to help them understand what is on the screen. This [Assistive Technology Glossary](https://makeitfable.com/glossary/) has many more examples.

**How is assistive tech related to `alt="sedan icon"?`**
A screen reader (which is assistive tech) will read out the text inside the alt attribute followed by the type of element. So it reads out "sedan icon image".

**And what would VoiceOver/JAWS interact?**
I am not sure I understand your question. VoiceOver/Jaws are screen reader programs that some people use to navigate and understand what's on the screen. This also applies to what is on a website.

When creating websites or web apps it's important to be aware of the people who will use them. I'm an advocate for creating a more inclusive experience by optimising for screen readers, respecting user preferences (font size, reduced motion), etc.

Web accessibility is an important topic - I recommend reading this short article [Introduction to Web Accessibility](https://a11y.coffee/intro/)